### PR TITLE
Make sidebar white

### LINKED ISSUE #22221: We should cut the "Navigation bar color" option from /admin/settings/whitelabel
The new redesign of Metabase's UI removes the old top blue bar, which was what the "Navigation bar color" option affected. I.e., once the new nav ships, this whitelabeling setting won't do anything

image

### PR DESCRIPTION
**Pull request by gusaiani**
Sidebar has been off-white for a couple of days.

This makes it white.

Part of fix #22221

### CODE REVIEW

File: frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.styled.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const Sidebar = styled.aside<{ isOpen: boolean }>`
   flex-shrink: 0;
   align-items: center;
   padding: 0.5rem 0;
+  background-color: white;
```
Comment (alxnddr) [inline frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.styled.tsx:33]: Can you use `color("white")` here?

Comment (gusaiani) [reply frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.styled.tsx]: Sure, done 👍

**Review by alxnddr — APPROVED** [review]

**Comment by flamber** [comment]
There was a couple of PRs addressing the sidebar color, so it would likely require a manual backport containing all the PRs.

**Comment by gusaiani** [comment]
@metabase-bot backport

**Pull request merged**